crates/sui-core/src/traffic_controller/policies.rs

```diff
@@ -49,6 +67,7 @@ impl TrafficSketch {
         sketch_capacity: usize,
         sketch_probability: f64,
         sketch_tolerance: f64,
+        highest_rates_capacity: usize,
```

Comment:
is this not yet called from anywhere? what sort of number did you have in mind for the capacity?